colour/utilities/common.py

```diff
@@ -1427,3 +1467,43 @@ def slugify(object_: Any, allow_unicode: bool = False) -> str:
     value = re.sub(r"[^\w\s-]", "", value.lower())
 
     return re.sub(r"[-\s]+", "-", value).strip("-_")
+
+
+if is_xxhash_installed():
+    import xxhash
+    from colour.utilities.documentation import is_documentation_building
+
+    int_digest = xxhash.xxh3_64_intdigest  # pyright: ignore
+
+    if is_documentation_building():  # pragma: no cover
+        import array
+
+        def int_digest(
+            args: str  # noqa: ARG001
+            | bytes
+            | bytearray
+            | memoryview
+            | array.ArrayType[int],
+            seed: int = 0,  # noqa: ARG001
+        ) -> int:
+            """
+            Generate an integer digest for given argument using *xxhash* if
+            available or falling back to :func:`hash` if not.
+
+            Parameters
+            ----------
+            args
+                Argument to generate the int digest of.
+            seed
+                Seed used to alter result predictably.
+
+            Returns
+            -------
+            :class:`int`
+                Integer digest.
+            """
+
+            return -1
+
+else:
+    int_digest = hash  # pyright: ignore
```

Comment:
Does this line need `#pragma: no cover`?

Otherwise I don't see any issues and this looks ready to merge.